components/ServiceList/index.js

```diff
@@ -0,0 +1,42 @@
+import useSWR from "swr";
+import Image from "next/image";
+
+export default function ServiceList() {
+  const { data, isLoading } = useSWR("/api/services");
```

Comment:
Right now, the code only checks for the absence of data to determine if there was an error. 
However, the absence of data could be due to various reasons. 

Ideally, you should check for explicit errors returned by useSWR.

```suggestion
  const { data, isLoading, error} = useSWR("/api/services");
```
-----
>💡 Additionally you can customize the [fetcher](https://github.com/ohlunlacassi/hotel-tishjama/pull/7/files?diff=unified&w=0#diff-c85e043f1db00b919581c615d87508f7d4ff2bb8058798ebbe475b4a6e087f26:~:text=const%20fetcher%20%3D%20(url)%20%3D%3E%20fetch(url).then((response)%20%3D%3E%20response.json())%3B) to throw an Error with additional information. 
Feel free to write this task on a small refactor list and take it on in a later refactor story. Dieses Thema wird im Handout 🔗 [React Data Fetching](https://github.com/neuefische/bo-web-23-4/blob/main/sessions/react-data-fetching/react-data-fetching.md) behandelt

Reply:
We are leaving this conversation "open" (but not unresolved), so that we might refer back to it later! Please advise if we should do something other than what we have just suggested.